website/docs: improve the docs about our Events and logging (#15270)
goauthentik/authentik: website/docs/sys-mgmt/events/transports.md

> title: Transports

Reviewer: The filename also needs to be updated to `notification_transports.md`

```suggestion
title: Notification Transports
```

Author: Let's hold off doing this (to avoid adding redirects) and see if the changes to the title and the sidebar are enough.

---

> To receive notifications about events, you will need to [create](#create-a-transport) a transport object, then create a notification rule and a policy. For details refer to [Workflow overview](./notifications.md#workflow-overview).

Reviewer 2: I think you prefered using must in these contexts?

> To receive notifications about events, you will need to [create](#create-a-transport) a transport object, then create a notification rule and a policy. For details refer to [Workflow overview](./notifications.md#workflow-overview).

Author: I do prefer "you must" though maybe it goes against our "be nice to the reader" rule. Let's discuss one day.

> To receive notifications about events, you must first [create a transport](#create-a-transport), then define a notification rule and a policy. For more information, see the [Workflow overview](./notifications.md#workflow-overview).

Reviewer: ```suggestion
To receive notifications about events, you must first [create a notification transport](#create-a-notification-transport), then define a notification rule with a bound policy. For more information, see the [Workflow overview](./notifications.md#workflow-overview).
```

> ## Transport modes

Reviewer: ```suggestion
## Notification transport modes
```

> Notifications can be sent to users via multiple mediums, or _transports_:

Reviewer: ```suggestion
Notifications can be sent to users via multiple mediums, or _notification transports_:
```

Author: Hmmm I feel like that is duplicative, snce we already say these are about notifications (forst word in sentence)... it is ore about explaining what a transport is...

- Local (in the authentik user interface)
- Email
- Webhook (generic)
- Webhook (Slack/Discord)

> ### Local transport

Reviewer: ```suggestion
### Local
```

> This transport will manifest the notification within the authentik user interface (UI).

Reviewer 2: ```suggestion
This transport will manifest the notification within the authentik User interface.
```

or if we want to become a little more special:

```suggestion
This transport will deliver the notification within the authentik user interface.
```

I don't like this one, but if you do 🤷

Author: in this case, I really do mean the "user interface" lower-case, meaning both Admin and User interfaces, the whole product... and thanks for letting me get away with using "manifest".

> This transport will manifest the notification within the authentik user interface (UI).

Reviewer: ```suggestion
This notification transport will manifest the notification within the authentik user interface (UI).
```

### Email

> Select this transport to send event notifications to an email address. Note that by default, the [global email configuration](../../install-config/install/docker-compose.mdx#email-configuration-optional-but-recommended) is used.

Reviewer 2: Is there any generic way of linking this rather than pointing to the compose insta;;

> To edit an email address, follow the same instructions as above, those for configuring the email during installation.

Reviewer 2: I'm not sure I understand the end of the sentence

The `Content-Type` header is set to `text/json`.

You can also select a Notification mapping. This allows you to freely configure the request's payload. For example:

```python
return {
    "foo": request.context['notification'].body,
}
```

### Webhook (Slack or Discord)

This sends a request using the Slack-specific format. This is also compatible with Discord's webhooks by appending `/slack` to the Discord webhook URL.

> ## Create a transport

Reviewer: ```suggestion
## Create a notification transport
```

1. Log in as an administrator to the authentik Admin interface, and navigate to **Event > Notification Transports**.

2. Click **Create** to add a new transport, or click the **Edit** icon next to an existing transport to modify it.

> 3. Define the **Name** and **Mode** for the transport, enter required configuration settings, and then click **Create**.

Reviewer: ```suggestion
3. Define the **Name** and **Mode** for the notification transport, enter required configuration settings, and then click **Create**.
```
